### PR TITLE
fix(c): Prefix import module name for resource-drop with `[export]`

### PR DESCRIPTION
**Pull request by kateinoigakukun**
Import module name for exported resource's drop should be prefixed with [export]

### CODE REVIEW

**Review by alexcrichton — APPROVED** [review]
Thanks! Mind updating the host-side of the test you're modifying here to call the new function as well?

**Comment by kateinoigakukun** [comment]
Sure!

**Comment by alexcrichton** [comment]
I think that this dotnet generator may need to exclude this test now, but otherwise looks good

**Comment by kateinoigakukun** [comment]
I don't have enough hardware access to fix c-sharp generator, so let me keep it disabled for now :cry:

**Review by alexcrichton — APPROVED** [review]

File: tests/runtime/resources/wasm.cs [hunk 1]
```diff
@@ -10,6 +10,12 @@ public static IExports.Z Add(IExports.Z a, IExports.Z b)
         {
             return new Z(((Z) a).val + ((Z) b).val);
         }
+
+        public static void Consume(IExports.X x)
+        {
+          // FIXME: c-sharp generator seems wrong here
+          // x.Dispose();
```
Comment (kateinoigakukun) [inline tests/runtime/resources/wasm.cs:17]: CC: @dicej There may be still resource GC hole around here in C# generator

**Pull request merged**